site/config.toml

```
@@ -80,9 +80,9 @@ github_repo = "https://github.com/googleforgames/agones"
 gcs_engine_id = "016691298986124624340:x7qv2dywdao"
 
 # current release branch. Never is rc.
-release_branch = "release-1.52.0"
+release_branch = "release-1.52.1"
```

```suggestion
release_branch = "release-1.52.0"
```

We're using the release-1.52.0 branch, and not creating a new one, so this should stay the same.